packages/dashboard/src/customization/widgets/lineSymbol/plugin.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import LineWidgetComponent from './component';
+import LineIcon from './icon';
+import type { DashboardPlugin } from '~/customization/api';
+import { LineWidget } from '../types';
+
+export const linePlugin: DashboardPlugin = {
+  install: ({ registerWidget }) => {
+    registerWidget<LineWidget>('line', {
+      render: (widget) => <LineWidgetComponent {...widget} />,
+      componentLibrary: {
+        name: 'Line',
+        icon: LineIcon,
+      },
+      properties: () => ({
+        lineStyle: 'solid',
```

Comment:
super nit: probably worth creating a const for this value object

Reply:
i looked at all other dashboard widgets and it's done this way. any merit to creating a const here?